src/main/java/dev/morling/kccli/command/GetLoggersCommand.java

```diff
@@ -0,0 +1,80 @@
+package dev.morling.kccli.command;
+
+import java.util.Iterator;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.freva.asciitable.AsciiTable;
+import com.github.freva.asciitable.Column;
+import com.github.freva.asciitable.HorizontalAlign;
+
+import dev.morling.kccli.service.KafkaConnectApi;
+import dev.morling.kccli.util.ConfigurationContext;
+import picocli.CommandLine;
+
+import static dev.morling.kccli.util.Colors.*;
+
+@CommandLine.Command(name = "loggers", description = "Get LOG Levels of passed connector/All if nothing is passed")
+public class GetLoggersCommand implements Runnable {
+
+    @Inject
+    ConfigurationContext context;
+
+    @CommandLine.Option(names = { "-p", "--path" }, description = "Path of the connector", defaultValue = DEFAULT_PATH)
```

Comment:
What's the "path" of a connector?

Reply:
Connector path is usually the root path of the connector based on the java package name. Or it can a single class name. I am ok to change the name if path doesnt seem correct.

Reply:
> Hey @sfc-gh-japatel, thanks for the update! Could you run `mvn license:format` and commit the result (currently, the license headers are missing from the new files)? Could you also update the output of _kcctl help_ in the README, showing the new command? Other than that, are you planning more changes, or is it good to go from your PoV?

Thanks @gunnarmorling 
Ran the mvn command. 
Added patch in help section of kcctl. 

These are the only changes I am planning from loggers perspective.